dev-docs/bidders/criteo.md

```diff
@@ -109,6 +109,18 @@ var adUnits = [
 
 ### Additional Config (Optional)
 
+If you don't want to use the FastBid adapter feature, you can disable it via this configuration:
```

Comment:
This seems unrelated

Reply:
Good catch! Some branches hick-up. Fixed now.